interchaintest/chain_upgrade_test.go

```diff
@@ -30,9 +30,9 @@ func TestPersistenceUpgradeBasic(t *testing.T) {
 	var (
 		chainName            = "persistence"
 		upgradeRepo          = PersistenceCoreImage.Repository
-		initialVersion       = "v11.5.2"
+		initialVersion       = "v11.6.0"
```

Comment:
not changing this?

Reply:
this is mentioning the outdated codebase.

was fixed in https://github.com/persistenceOne/persistenceCore/pull/295/commits/0f8992ee741b3f9745aec10ef5ef5a65df4b8190 above